values-secret.yaml.template

```diff
@@ -3,4 +3,25 @@ secrets:
   # NEVER COMMIT THESE VALUES TO GIT
   config-demo:
     # Secret used for demonstrating vault storage, external secrets, and ACM distribution
-    secret: PLAINTEXT
+    secret: VALUE
+
+  # Required for automated spoke deployment
+  aws:
+      access_key_id: VALUE
+      secret_access_key: VALUE
+
+# Required for automated spoke deployment
+# DO NOT use ~/ , it will appear to work be the secrets will be empty
```

Comment:
This has been fixed via https://github.com/hybrid-cloud-patterns/common/commit/8f03fa648d300adec315e0fb3cdb4afc4d734014 so let's remove this line?

Reply:
```suggestion
```